definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -26,14 +26,16 @@
     "SecurityClassification": "Public"
   },
   {
-    "LiveFrom": "01/01/2017",
+    "LiveFrom": "04/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "servicePayment",
-    "Name": "Service payment exists",
-    "Description": "Service payment exists",
-    "DisplayOrder": 2,
-    "PreConditionState(s)": "AwaitingServicePayment",
+    "ID": "ConfirmServicePayment",
+    "Name": "Confirm service payment",
+    "Description": "Confirm service payment",
+    "DisplayOrder": 13,
+    "PreConditionState(s)": "AwaitingPayment",
     "PostConditionState": "AwaitingServiceConsideration",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/set-up-confirm-service-payment-event",
+    "RetriesTimeoutURLAboutToSubmitEvent": "30,30",
```

Comment:
I don't know why you always use "30,30" if for most of the cases it's 120,120?

To be honest I don't know which one is better. I don't even know if it's really implemented, but I'd rather use option that is used (nearly) everywhere else.

Reply:
14 matches for "30,30"
>100 for "120,120".

Please use 120,120, unless you used "30,30" for a good reason.